src/app/modules/main/stickers/models/sticker_pack_list.nim

```diff
@@ -0,0 +1,141 @@
+import NimQml, Tables, sequtils, sugar
+import ./sticker_list
+import status/utils as status_utils
```

Comment:
@jrainville same as above, don't rely on anything from status-lib except methods which do status-go API calls (this and a line below).